nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/coordinator/WriteCoordinator.scala

```diff
@@ -378,31 +463,51 @@ class WriteCoordinator(metadataCoordinator: ActorRef, schemaCoordinator: ActorRe
         updateSchema(db, namespace, metric, bit) { schema =>
           val consistentLocations = locations.take(consistencyLevel)
           val eventualLocations   = locations.diff(consistentLocations)
+          //FIXME common code
```

Comment:
please evaluate if this refactor is simple enough to be performed in this PR. If you think this deserve a dedicated PR or it can be done in another PR, just tell it here.